src/messagingsession/MessagingSessionConfiguration.ts

```diff
@@ -27,6 +29,11 @@ export default class MessagingSessionConfiguration {
    */
   reconnectLongBackoffMs: number = 5 * 1000;
 
+  /**
+   * The enum to indicate if we want to turn on prefetch feature
```

Comment:
Update the comment. It is not whether we turn it on right but rather when to turn it on (which is right now just on reconnect)

Reply:
Hi Trung, this is not specific for reconnect. This is a new feature for connect in general.

Reply:
This documentation needs update, Trung mentioned that we want to explain more what prefetch feature means.

Reply:
Need more details about prefetch feature.